presto-hive/src/main/java/com/facebook/presto/hive/s3/PrestoS3FileSystem.java

```diff
@@ -713,6 +719,10 @@ private AWSCredentialsProvider getAwsCredentialsProvider(URI uri, Configuration
             return InstanceProfileCredentialsProvider.getInstance();
         }
 
+        if (iamRole != null) {
+            return new STSAssumeRoleSessionCredentialsProvider.Builder(this.iamRole, "presto-session").build();
```

Comment:
Looks like it's possible to log the role session name to aws cloudtrail logs (https://docs.aws.amazon.com/STS/latest/APIReference/API_AssumeRole.html), so do you think it will be valuable to specify maybe something like `"presto - " + uri` as the `roleSessionName` here instead of `presto-session` to also have the accessed uri as part of the role name? I never had to debug through role names and cloudtrail logs before, but if you think it's valuable we can add that.